mobile/lib/features/trade/channel_configuration.dart

```diff
@@ -38,27 +38,21 @@ channelConfiguration({
       context: context,
       useSafeArea: true,
       builder: (BuildContext context) {
-        return SafeArea(
-          child: Container(
-            padding: EdgeInsets.only(bottom: MediaQuery.of(context).viewInsets.bottom),
-            child: GestureDetector(
-                onTap: () {
-                  FocusScopeNode currentFocus = FocusScope.of(context);
-
-                  if (!currentFocus.hasPrimaryFocus) {
-                    currentFocus.unfocus();
-                  }
-                },
-                child: SingleChildScrollView(
-                  child: SizedBox(
-                    height: 500,
-                    child: ChannelConfiguration(
-                      tradeValues: tradeValues,
-                      onConfirmation: onConfirmation,
-                    ),
-                  ),
-                )),
-          ),
+        return Wrap(
```

Comment:
🤔 Are you sure this will work when toggling the keyboard?

Reply:
Good question, let me try.